### PR TITLE
fix: APPS-2480 Update contact email on SMDL

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-2480](https://uclalibrary.atlassian.net/browse/APPS-2480)

Update contact email on SMDL

[APPS-2480]: https://uclalibrary.atlassian.net/browse/APPS-2480?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/static/terms_of_use.html.erb [hunk 1]
```diff
@@ -41,7 +41,7 @@
...
-        <p class='static-page__text static-page__text--sinai'>Users are invited to review the <%= link_to 'Privacy Policy', 'https://sinai-id.org/static_pages/spp_privacy', target: '_blank', rel: 'noopener', class: 'si-link' %> associated with their login. Registration data is managed by the Early Manuscripts Electronic Library (EMEL). If you ever wish to delete your user account, email a request to <%= mail_to 'sinai.id.emel@gmail.com', 'sinai.id.emel@gmail.com', class: 'si-link' %>, and EMEL will remove your account data from its database.</p>
+        <p class='static-page__text static-page__text--sinai'>Users are invited to review the <%= link_to 'Privacy Policy', 'https://sinai-id.org/static_pages/spp_privacy', target: '_blank', rel: 'noopener', class: 'si-link' %> associated with their login. Registration data is managed by the Early Manuscripts Electronic Library (EMEL). If you ever wish to delete your user account, email a request to <%= mail_to 'sinai@library.ucla.edu', 'sinai@library.ucla.edu', class: ' si-link' %>, and EMEL will remove your account data from its database.</p>
```
Comment (pghorpade) [inline app/views/static/terms_of_use.html.erb:44]: The ticket does not say remove sinai emel id email address

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: app/views/static/terms_of_use.html.erb (hunk 1 above)
Comment (kirschbombe) [reply app/views/static/terms_of_use.html.erb:44]: confirming that we want to leave the sinai.id.emel@ emails in place :) thanks for catching that!

Comment (jendiamond) [reply app/views/static/terms_of_use.html.erb]: fixed

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**